demo/src/app/components/+pagination/demos/index.ts

```diff
@@ -2,10 +2,12 @@ import { DemoPaginationBasicComponent } from './basic/basic';
 import { DemoPaginationPagerComponent } from './pager/pager';
 import { DemoPaginationLimitComponent } from './limit/limit';
 import { DemoPaginationStylingComponent } from './styling-global/styling-global';
+import { DemoPaginationStatesComponent } from './states/states';
```

Comment:
`states` says nothing about disabled state. `DemoPaginationDisabledComponent` would be better